docs/cookbooks/async_streaming.ipynb

```diff
@@ -0,0 +1,351 @@
+{
+    "cells": [
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "# Async Stream-validate LLM responses\n"
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+       "# Few imports and global variables\n",
+       "from rich import print\n",
+       "import guardrails as gd\n",
+       "import litellm\n",
+       "from IPython.display import clear_output\n",
+       "import time"
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "### Setup\n",
+       "\n",
+       "Install the necessary validators from Guardrails hub in your CLI.\n",
+       "\n",
+       "```bash\n",
+       "!guardrails hub install hub://guardrails/valid_range\n",
+       "!guardrails hub install hub://guardrails/uppercase\n",
+       "!guardrails hub install hub://guardrails/lowercase\n",
+       "!guardrails hub install hub://guardrails/one_line\n",
+       "```"
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "### 1. For structured JSON output\n"
```

Comment:
Am I mistaken, or is the streaming implementation for JSON schema still the old one? Should we highlight it so prominently in our docs?